### PR TITLE
Update pvactools CWLs to version 2.0.0

### PR DESCRIPTION
**Pull request by susannasiebert**
This involved quite a few changes to the inputs since those changed in the new version. The VEP version was also updated to one that has the new Frameshift plugin, which is required for running pVACseq 2.0.

@johnegarza if you could have a look at the immuno.cwl to make sure I didn't miss anything, that would be great.

### CODE REVIEW

File: definitions/tools/pvacbind.cwl [hunk 1]
```diff
@@ -44,16 +44,26 @@ inputs:
...
         inputBinding:
             position: 4
-    epitope_lengths:
+    epitope_lengths_class_i:
         type: int[]?
         inputBinding:
-            prefix: "-e"
+            prefix: "-e1"
+            itemSeparator: ','
+            separate: false
```
Comment (tmooney) [inline definitions/tools/pvacbind.cwl:52]: With 
```cwl
separate: false
```
an input like
```yaml
epitope_lengths_class_i: [5,6]
```
would become
```bash
-e15,6,7
```
in the command line.  I see that `separate: false` is in the old version, but should we take it out for this (and the class_ii input below it)?  (Repeat this comment for other tools with these inputs, as well.)

File: definitions/tools/pvacfuse.cwl [hunk 2]
```diff
@@ -47,16 +47,26 @@ inputs:
...
         type: int[]?
         inputBinding:
-            prefix: "-e"
+            prefix: "-e1"
+            itemSeparator: ','
+            separate: false
+    epitope_lengths_class_ii:
+        type: int[]?
+        inputBinding:
+            prefix: "-e1"
```
Comment (tmooney) [inline definitions/tools/pvacfuse.cwl:59]: Looks like a typo here.
```suggestion
            prefix: "-e2"
```

Comment (susannasiebert) [reply definitions/tools/pvacfuse.cwl]: Fixed.

File: definitions/tools/pvacbind.cwl (hunk 1 above)
Comment (susannasiebert) [reply definitions/tools/pvacbind.cwl]: I updated all the tool CWLs to remove the separate option

**Review by tmooney — DISMISSED** [review]
+1 🦝

**Review by johnegarza — APPROVED** [review]

**Comment by johnegarza** [comment]
+1 immuno wf looks good!

**Comment by gschang** [comment]
+1 These updates with new pVACtools 2.0.0 version are exciting. Jasreet, Mike, and I are discussing to use this version for the clinical studies. Nice presentation today.

**Pull request merged**